README.md

```diff
@@ -216,7 +288,7 @@ A `ThirdPartyNotices.txt` file is provided in the extension's source code listin
 - The first time you install the extension, you'll need to execute the `run` command at least once in order to access auto-completion.
 - While running a code file, if you get an error saying it can't find the file, make sure you've clicked on a valid Python code file before running it.
 - To open the output panel again after closing it go to VS Code menu: `View->Output`.
-- If you try to deploy to the device while it's plugged in but you still get an error saying it cannot find the board, make sure your Circuit Playground Express is formatted correctly and that its name matches `CIRCUITPY`.
+- If you try to deploy to the Circuit Playground Express while it's plugged in but you still get an error saying it cannot find the board, make sure your device is formatted correctly and that its name matches `CIRCUITPY`.

```

Comment:
Inconsistent use of Circuit Playground Express. Ideally, it should probably just be "CPX" everywhere except the first mention.

Reply:
Valid point. I changed it in the places I see fit. Let me know :D